### PR TITLE
style: select arrow won't rotate now

### PR DESCRIPTION
**Pull request by afc163**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Select arrow won't rotate when open. |
| 🇨🇳 Chinese | Select 箭头打开时不再翻转。 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/select/style/index.less [hunk 1]
```diff
@@ -99,7 +99,7 @@
   &-arrow {
     .iconfont-mixin();
     position: absolute;
-    top: 53%;
+    top: 50%;
```
Comment (xrkffgg) [inline components/select/style/index.less:102]: 我感觉 53 % 的 更协调一些 😂

Comment (afc163) [reply components/select/style/index.less:102]: 截个图来对比一下？

**Comment by xrkffgg** [comment]
# 1
image

# 2
image

@afc163 @yoyo837 看下  1、2 哪个更协调？

**Comment by xrkffgg** [comment]
@zombieJ @AshoneA

**Comment by yoyo837** [comment]
2

**Comment by xrkffgg** [comment]
## 先选择再看答案哦

<details>
<summary>答 案</summary>

2 是 53 %

</details>

**Comment by afc163** [comment]
先保持吧 53% 吧。

**Review by xrkffgg — APPROVED** [review]

**Comment by zombieJ** [comment]
哈哈，原来已经做掉了。怪不得找了半天没发现旋转。

**Comment by ghostlhq** [comment]
为啥要删了呢嘤嘤嘤

**Pull request merged**